tests/common/helpers/platform_api/psu.py

```diff
@@ -117,3 +120,10 @@ def get_all_thermals(conn, psu_id):
 
 def get_thermal(conn, psu_id, index):
     return psu_api(conn, psu_id, 'get_thermal', [index])
+
```

Comment:
Per [PEP8](https://www.python.org/dev/peps/pep-0008/), there should be two blank lines between top-level function definitions.